### PR TITLE
feat: rating

### PR DESCRIPTION
**Pull request by shashidwivedi**
Rating component

- rating filled
- rating disabled
- no rating

image

### CODE REVIEW

**Review by shivangisharma01 — APPROVED** [review]
Looks good to merge, no changes required.

File: styles/foobar-ui.css [hunk 1]
```diff
@@ -751,7 +755,17 @@ template {
   border: 1px solid var(--color-success);
```
Comment (vaibhav-mahalle) [inline styles/foobar-ui.css:755]: liked the use of variables for colors.

File: styles/foobar-ui.min.css [hunk 2]
```diff
@@ -1 +1 @@
...
...md)}.rating .rating-star{cursor:pointer}.rating .icon-filled{color:var(--color-secondary)}.heading-1{font-size:var(--font-size-xl);line-height:var(--line-height-md)}.heading-2{font-size:var(--font-size-lg);line-height:var(--line-height-sm)}.heading-3{font-size:var(--font-size-md);line-height:var(--line-height-md)}.heading-4{font-size:var(--font-size-sm);line-height:var(--line-height-lg)}.heading-5{font-size:var(--font-size-xs);line-height:var(--line-height-xl)}.text-1{font-size:var(--font-size-xl);line-height:var(--line-height-md)}.text-2{font-size:var(--font-size-lg);line-height:var(--line-height-sm)}.text-3{font-size:var(--font-size-md);line-height:var(--line-height-md)}.text-4{font-size:var(--font-size-sm);line-height:var(--line-height-lg)}.text-5{font-size:var(--font-size-xs);line-height:var(--line-height-xl)}.text-small{font-size:calc(var(--font-size-sm) * 5 / 6)}.text-gray{color:var(--color-gray)}.text-center{text-align:center}.text-black{font-weight:var(--font-weight-black)}.text-bold{font-weight:var(--font-weight-bold)}.text-semibold{font-weight:var(--font-weight-semibold)}.text-regular{font-weight:var(--font-weight-regular)}.text-light{font-weight:var(--font-weight-light)}
```
Comment (vaibhav-mahalle) [inline styles/foobar-ui.min.css:1]: please look into the indentation of the file.

**Review by vaibhav-mahalle — APPROVED** [review]
suggested some changes please look into it.

File: styles/foobar-ui.min.css (hunk 2 above)
Comment (shashidwivedi) [reply styles/foobar-ui.min.css:1]: Hi Vaibhav, it's a generated CSS minified file. No need for indentation here. Thanks for the suggestion!

**Pull request merged**